server/push/tnpg/push_tnpg.go

```diff
@@ -23,10 +23,11 @@ type Handler struct {
 }
 
 type configType struct {
-	Enabled          bool `json:"enabled"`
-	Buffer           int  `json:"buffer"`
-	CompressPayloads bool `json:"compress_payloads"`
-	BatchSize        int  `json:"batch_size"`
+	Enabled          bool   `json:"enabled"`
+	Buffer           int    `json:"buffer"`
+	CompressPayloads bool   `json:"compress_payloads"`
+	BatchSize        int    `json:"batch_size"`
```

Comment:
As I wrote in the other PR I would make this hardcoded 100.

Reply:
This is an outdated snapshot. I removed it.